saleor/product/models.py

```diff
@@ -238,11 +238,13 @@ def display_variant(self, attributes=None):
                 ['%s: %s' % (smart_text(attributes.get(id=int(key))),
                              smart_text(value))
                  for (key, value) in values.items()])
-        else:
-            return smart_text(self.sku)
+        return ''
```

Comment:
This method will return anything only if there are attributes. I think we should rename it to `display_variant_attributes` to make it clear when to use it.